rhp/v4/server.go

```diff
@@ -585,14 +585,14 @@ func (s *Server) handleRPCFormContract(stream net.Conn) error {
 
 	// update renter input basis to reflect our funding basis
 	if basis != req.Basis {
-		hostInputs := formationTxn.SiacoinInputs[len(formationTxn.SiacoinInputs)-len(req.RenterInputs)]
-		formationTxn.SiacoinInputs = formationTxn.SiacoinInputs[:len(formationTxn.SiacoinInputs)-len(req.RenterInputs)]
+		hostInputs := formationTxn.SiacoinInputs[len(req.RenterInputs):]
```

Comment:
This will likely need to be fixed in RPC refresh and renew

Reply:
Yeah I know. Already working on it.